specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2025-03-01-preview/PreviewAlertRule.json

```diff
@@ -0,0 +1,627 @@
+          "type": "string",
+          "description": "Name of the dimension"
+        },
+        "operator": {
+          "type": "string",
+          "description": "Operator for dimension values",
+          "enum": [
+            "Include",
+            "Exclude"
+          ],
+          "x-ms-enum": {
+            "name": "dimensionOperator",
+            "modelAsString": true
+          }
+        },
+        "values": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "List of dimension values"
+        }
+      },
+      "required": [
+        "name",
+        "operator",
+        "values"
+      ]
+    },
+    "LogAlertRuleIdentity": {
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The principal ID of resource identity."
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant ID of resource."
+        },
+        "type": {
+          "type": "string",
+          "description": "Type of managed service identity.",
+          "enum": [
+            "SystemAssigned",
+            "UserAssigned",
+            "None"
+          ],
+          "x-ms-enum": {
+            "name": "IdentityType",
+            "modelAsString": false
+          }
+        },
+        "userAssignedIdentities": {
+          "description": "The list of user identities associated with the resource. The user identity dictionary key references will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}'.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/LogAlertRuleUserIdentityProperties"
+          }
+        }
+      },
+      "required": [
+        "type"
+      ],
+      "type": "object",
+      "description": "Identity for the resource."
+    },
+    "LogAlertRuleUserIdentityProperties": {
+      "type": "object",
+      "description": "User assigned identity properties.",
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "description": "The principal id of user assigned identity.",
+          "type": "string"
+        },
+        "clientId": {
+          "readOnly": true,
+          "description": "The client id of user assigned identity.",
+          "type": "string"
+        }
+      }
+    },
+    "PreviewAlertRuleResponse": {
+      "type": "object",
+      "properties": {
+        "rulePreviewResults": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RulePreviewResult"
+          },
+          "description": "An array of rule preview results."
+        }
+      }
+    },
+    "RulePreviewResult": {
+      "type": "object",
+      "properties": {
+        "dimensionCombinations": {
```

Comment:
These properties must have id,name,type, properties as the top level properties. dimensionCombinations and any other properties must be pushed inside the properties bag